mise.toml

```diff
@@ -14,7 +14,7 @@ shellcheck = "0.8.0"
 yq = "4.47.1"
 "aqua:bufbuild/protoc-gen-validate" = "1.2.1"
 "aqua:grpc/grpc-go/protoc-gen-go-grpc" = "1.5.1"
-"aqua:helm/chart-releaser" = "1.7.0"
+"aqua:helm/chart-releaser" = "1.8.1"
```

Comment:
should we run this locally (release to fork) to see this version works with our release scripts?